FAIL
run: ((null)) should have called exit(-1): FAILED
